compiler/optimizer/OMROptimizer.cpp

```diff
@@ -1501,7 +1501,7 @@ int32_t OMR::Optimizer::performOptimization(const OptimizationStrategy *optimiza
 
       case IfMethodHandleInvokes:
```

Comment:
This change seems to change the intuitive meaning of IfMethodHandleInvokes - should the name of the case change to reflect the change in the logic underneath?

Reply:
I think `IfMethodHandleInvokes` still means the optimizations are specific to method handle invokes but just the way to detect method handle invoke is changed. Before, the detection happens when doing ilgen and now it happens after inlining.  We could just use `IfEnabled` and get rid of `IfMethodHandleInvokes` though.

Reply:
I would rather switch to IfEnabled in the opt strategy and preserve the meaning of IfMethodHandleInvokes because it maintains the semantic meaning of the ifs.

Reply:
Changed `IfMethodHandleInvokes` to `IfEnabled` in opt strategy as suggested. Keep the `IfMethodHandleInvokes` case in `performOptimization`  unchanged to preserve the real meaning of `IfMethodHandleInvokes`.